pom.xml

```diff
@@ -306,14 +306,14 @@
         <artifactId>maven-javadoc-plugin</artifactId>
         <version>${commons.javadoc.version}</version>
         <configuration>
-          <!-- keep only errors and warnings -->
-          <show>private</show>
+          <source>${maven.compiler.source}</source>
           <quiet>true</quiet>
-          <encoding>${commons.encoding}</encoding>
+          <encoding>utf-8</encoding>
+          <docEncoding>utf-8</docEncoding>
           <notimestamp>true</notimestamp>
           <links>
-            <link>${commons.javadoc.java.link}</link>
-            <link>${commons.javadoc.javaee.link}</link>
+            <link>https://docs.oracle.com/javase/8/docs/api/</link>
+            <link>http://docs.oracle.com/javaee/6/api/</link>
```

Comment:
I think these values must not be altered here. They are specified in the `commons-parent` module ([link](https://github.com/apache/commons-parent/blob/399b9eeb66d3b8cb030c9dd12099251b94a7f79a/pom.xml#L278))